Classes/commands.py

```diff
@@ -853,6 +852,59 @@ async def show_loa(self, ctx):
             string = "There are no Leaves of Absence on file at this time."
         await ctx.channel.send(string)
 
+    @checks.is_admin_bot_channel()
+    @checks.is_white_shirt()
+    @commands.command()
+    async def accept_inactive_reasons(self, ctx):
+        """Approve all inactivity reasons listed in the mentioned channels"""
+        inactive_role = self.bot.officer_manager.guild.get_role(
+            self.bot.settings["inactive_role"]
+        )
+
+        for channel in ctx.message.channel_mentions:
+            async for message in channel.history(limit=None):
+                if inactive_role in message.author.roles:
+                    await message.author.remove_roles(inactive_role)
+                    officer = self.bot.officer_manager.get_officer(message.author.id)
+                    await officer.renew_time()
+
+        unexcused_mentions = ""
+        for member in ctx.guild.members:
+            if inactive_role in member.roles:
+                unexcused_mentions = f"{unexcused_mentions}{member.mention}"
```

Comment:
This needs an enter between each line, since this list can be in the hundreds of names